Mathlib/Algebra/GCDMonoid/Div.lean

```diff
@@ -3,11 +3,11 @@ Copyright (c) 2022 Riccardo Brasca. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Riccardo Brasca
 -/
-import Mathlib.Algebra.GCDMonoid.Finset
 import Mathlib.Algebra.GCDMonoid.Basic
-import Mathlib.RingTheory.Int.Basic
-import Mathlib.RingTheory.Polynomial.Content
+import Mathlib.Algebra.GCDMonoid.Finset
+import Mathlib.Algebra.GCDMonoid.Int
 import Mathlib.Algebra.GCDMonoid.Nat
+import Mathlib.RingTheory.Polynomial.Content
```

Comment:
For a followup, this import (as any `RingTheory → Algebra` import) is suspicious. Shouldn't it go the other way around?

Reply:
I guess the
```
## TODO
Add a typeclass to state these results uniformly.
```
explains it. Do we have an idea what such a typeclass would look like? It seems all we want is `∀ a b, a ≠ 0 → a * b / a = b`, which is pretty easy to state.

Reply:
I have now opened #12424 to define said typeclass